### PR TITLE
1201 email accessability

### PR DESCRIPTION
**Pull request by EricDube**
This fixes the issue that Accessibility flag was not properly being sent to the email service. Since this code was using "FamilyCheck" I decided to remove all the Family related code from the source.

### CODE REVIEW

File: src/pages/ReviewPage.js [hunk 1]
```diff
@@ -33,25 +33,17 @@ class ReviewPage extends React.Component {
...
-        return <Trans>No</Trans>
-    }} else {
+    if (reason || reason === 'yes') {
+      return <Trans>Yes</Trans>
+    } else {
       return <Trans>No</Trans>
     }
   }
 
   translate(reason) {
```
Comment (MarcoGoC) [inline src/pages/ReviewPage.js]: translate is not used, we should just delete the code

File: src/email/handleSubmitEmail.js [hunk 2]
```diff
@@ -2,12 +2,14 @@ import { sendNotification } from './sendmail';
...
 const CONFIRMATION_EMAIL = 'af563da5-43bd-4b9a-9610-e3990a7f4315'
-  
+
 
 
 export const handleSubmitEmail = async (req, res) => {
   let input = Object.assign({}, req.body) // make a new object
   if (input.templateId === CONFIRMATION_EMAIL){
+    console.log("handleSubmitEmail Line 11:"+JSON.stringify(input))
+    console.log("handleSubmitEmail Line 12:"+input.accessibility)
```
Comment (andr0272) [inline src/email/handleSubmitEmail.js]: Is this not throwing an error with es-lint? This looks like debugging logs.

File: src/pages/ReviewPage.js (hunk 1 above)
Comment (EricDube) [reply src/pages/ReviewPage.js]: Removed!

File: src/pages/ConfirmationPage.js [hunk 3]
```diff
@@ -104,8 +99,7 @@ class ConfirmationPage extends React.Component {
           register: {
             paperFileNumber,
             email,
-            familyCheck,
-            familyOption,
+            accessibility,
             // hashFromData,
```
Comment (andr0272) [inline src/pages/ConfirmationPage.js]: >

File: src/validation.js [hunk 4]
```diff
@@ -84,8 +84,6 @@ export const defaultMessages = {
   //'required.explanation': 'explanationErrorMessage',
```
Comment (MarcoGoC) [inline src/validation.js]: We should delete all these comments too

File: src/pages/ConfirmationPage.js [hunk 5]
```diff
@@ -138,36 +132,30 @@ class ConfirmationPage extends React.Component {
...
             email={email}
-            accessibility={this.translateReason(familyCheck)}
-            privacy={this.translateReason(familyOption)}
-            location={
-              locationCity && locationAddress
-                ? locationCity + ', ' + locationAddress
-                : ''
-            }
+            accessibility={this.translateReason(accessibility)}
+            location={(locationCity && locationAddress) ? locationCity + ', ' + locationAddress : ''}
```
Comment (andr0272) [inline src/pages/ConfirmationPage.js]: Check out the back tick string syntax in es6 when you have a chance. Super clean.

File: src/email/handleSubmitEmail.js (hunk 2 above)
Comment (EricDube) [reply src/email/handleSubmitEmail.js]: Good catch!

File: src/pages/RegistrationPage.js [hunk 6]
```diff
@@ -83,15 +72,11 @@ const labelNames = id => {
     case 'emailConfirm':
       return <Trans>Confirm Email address</Trans>
     case 'accessibility':
-      return ''
+      return 'TODO: Input correct string'
```
Comment (MarcoGoC) [inline src/pages/RegistrationPage.js]: accessibility and privacy are the same, we should delete this case

File: src/pages/RegistrationPage.js [hunk 7]
```diff
@@ -110,19 +95,14 @@ class RegistrationPage extends React.Component {
 
   static validate(values, submitted) {
     let registrationFields = RegistrationFields
-    deleteEmptyArrayKeys(values)
+    //deleteEmptyArrayKeys(values) //TODO: What is this
```
Comment (MarcoGoC) [inline src/pages/RegistrationPage.js]: If the Checkbox or Radio buttons the original form used to have where empty(not selected) this function will remove the key from the values been passed to the validator.

Comment (EricDube) [reply src/pages/RegistrationPage.js]: Thats what I thought it did, we actually need the field to be there so for future checks. What was the reason we wanted to remove the key completely?

File: src/pages/RegistrationPage.js (hunk 6 above)
Comment (EricDube) [reply src/pages/RegistrationPage.js]: I removed Privacy and set the proper string, thanks for catching that!

**Pull request merged**